### ISSUE TITLE
Use of Enso IDE without an account

### ISSUE THREAD

**Issue by harry-xm** [enso-org/enso#10378]
### How important is this feature to you?

5 – Lack of it blocks me from doing my work in Enso

### Describe the idea you'd like to see implemented.

Implement offline mode for Enso IDE, which was removed in #9987 and the linked issue is not publicly accessible.

### Is your feature request related to a problem?

I would like to analyze data offline on my computer, but in the latest release, the "Continue without creating an account" option is gone.

### Would you be willing to help us implement this feature?

No

**Comment by somebody1234** [comment]
Analyzing data locally still works in the latest versions, you will just need to create an account - as long as you've logged in once, you *should* be able to use it fine even without internet access.

If you're concerned about something else (e.g. privacy issues) then it would be great if you could elaborate.

Also regarding offline mode - I think it *was* broken by that PR, but working offline should work again as of recently:
- #10317

**Comment by harry-xm** [comment]
> If you're concerned about something else (e.g. privacy issues) then it would be great if you could elaborate.

@somebody1234 Yes, I prefer local-first software[1] for my data analytics needs. My reasons are as follows:

1. Security and privacy by default. I'm not bound to the ToS and privacy policy of a cloud service, and reduced network attack vector is good for security.
2. Software longevity. Even if the cloud services experienced an outage or have upgraded its API to become incompatible with  older versions of the IDE, it would not break the functionality I use locally.
3. True offline mode. In an isolated network, or countries and regions where Internet connectivity is restricted, I don't have to circumvent the restrictions, which could be illegal or impossible.
4. Ownership and control of the software and my data. I can fork and customize the code with confidence that I won't be forced to merge new changes or workaround the cloud integration, and I can make complete backups of my code, data and the software.

[1]: https://martin.kleppmann.com/papers/local-first.pdf
